lib/jekyll-plugin-platoniq-journal/quote_tag.rb

```diff
@@ -3,7 +3,7 @@
 # require "byebug"
 
 module Jekyll
-  class QuoteBlockTag < Liquid::Block
+  class QuoteBlockTag < LocalizedBlock
```

Comment:
once we have the `Base` module we can include it, so the `QuoteBlockTag` can inherit from `Liquid::Block`as it was before:

```suggestion
  class QuoteBlockTag < Liquid::Block
    include JekyllPluginPlatoniqJournal::Base
```